pkg/kv/kvserver/replica_raft.go

```diff
@@ -1406,21 +1406,24 @@ func (r *Replica) refreshProposalsLocked(
 		r.breaker.TripAsync(err)
 	}
 
-	if log.V(1) && len(reproposals) > 0 {
-		log.Infof(ctx,
-			"pending commands: reproposing %d (at %d.%d) %s",
-			len(reproposals), r.mu.state.RaftAppliedIndex,
-			r.mu.state.LeaseAppliedIndex, reason)
+	if len(reproposals) == 0 {
+		return
 	}
 
+	log.VInfof(ctx, 1,
+		"pending commands: reproposing %d (at %d.%d) %s",
```

Comment:
Maybe make %d.%d more verbose on what this is, it's probably easy to get confused.

How about `RAI`/`LAI`, are these established abbreviations?